libraries/AP_Camera/AP_Camera.h

```diff
@@ -98,18 +98,18 @@ class AP_Camera {
     uint32_t        _last_photo_time;   // last time a photo was taken
     struct Location _last_location;
     uint16_t        _image_index;       // number of pictures taken since boot
-    uint16_t        _feedback_events;   // number of feedback events since boot
 
     // pin number for accurate camera feedback messages
     AP_Int8         _feedback_pin;
     AP_Int8         _feedback_polarity;
 
     // this is set to 1 when camera trigger pin has fired
-    static volatile bool   _camera_triggered;
+    uint32_t        _camera_trigger_count;
+    uint32_t        _camera_trigger_logged;
+    uint32_t        _feedback_timestamp_us;
     bool            _timer_installed:1;
+    bool            _isr_installed:1;
```

Comment:
Can we not add bit fields? It's been proven in some places to be worse for flash space rather than making it better.